src/config/texts.js

```diff
@@ -404,7 +404,29 @@ export const texts = {
     pushNotifications: 'Push-Benachrichtigungen'
   },
   survey: {
-    archive: 'Umfrage-Archiv'
+    archive: 'Umfrage-Archiv',
+    answerLabelPrefix: {
+      de: 'Antwort',
+      pl: 'Odpowiedź'
+    },
+    dateEnd: {
+      de: 'Abschluss der Umfrage:',
+      pl: 'Wypełnienie ankiety:'
+    },
+    dateStart: {
+      de: 'Start der Umfrage:',
```

Comment:
`Startdatum der Umfrage` in ticket

Reply:
as discussed in daily: `Start der Umfrage:` is what we want here